src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: calc(7 * $cell-size + 6 * $gap);
+  padding: $calendar-padding;
```

Comment:
Ensure that the variable `$calendar-padding` is defined in your SCSS files. If it's missing, it will cause a compilation error.